src/test/model_config_test_base.cc

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2018, NVIDIA CORPORATION. All rights reserved.
```

Comment:
Are you missing some of the "simpler" tests? Like what if an undefined ensemble tensor name is specified, what if a model input or output is not used, what if a tensor (ensemble or output) is assigned to multiple times, etc.